extensions/amp-story/1.0/pagination-buttons.css

```diff
@@ -71,13 +68,12 @@
   background-color: #fff!important;
   background-repeat: no-repeat!important;
   background-size: 12px 17px!important;
-  display: flex!important;
   justify-content: center!important;
   align-items: center!important;
   opacity: .5!important;
   transition: 150ms opacity linear, 350ms transform linear!important;
   cursor: pointer!important;
-  z-index: 100002!important;
```

Comment:
Is it intentional to drop the z-index?

Reply:
It's set on the parent element `.i-amphtml-story-button-container`.
I updated `Z_INDEX.md` to document this change.